lib/grammers-client/src/client/files.rs

```diff
@@ -21,6 +21,7 @@ use tokio::{
 
 pub const MIN_CHUNK_SIZE: i32 = 4 * 1024;
 pub const MAX_CHUNK_SIZE: i32 = 512 * 1024;
+pub const FILE_MIGRATE_ERROR: i32 = 303;
```

Comment:
The chunk size is public to let the users know that's the limits they can configure but this error doesn't really need to be exposed to the users:

```suggestion
const FILE_MIGRATE_ERROR: i32 = 303;
```